static/app/utils/theme/theme.chonk.tsx

```diff
@@ -309,7 +309,7 @@ function generateChonkTokens(colorScheme: typeof lightColors) {
     content: {
       primary: colorScheme.gray800,
       muted: colorScheme.gray500,
-      accent: colorScheme.blue500,
+      accent: colorScheme.blue300,
```

Comment:
the `activeHover` is `purple400` in the theme and `blue300` in chonk, so I just mapped the same.

Reply:
It's difficult to understand your intention here, without knowing what you're using it for.

These alias are meant for painting text and UI icons to reach an AA contrast ratio. If you change this to `blue300` you will be using a colour that's a very low-contrast transparent blue.

Reply:
I was just following the same logic as the activeHover property. I noticed it uses purple400  in the theme and blue300 in the chonk, applying the same approach in both places.